### PR TITLE
Add "Add Credit Card" examples for each language

### PR DESCRIPTION
**Pull request by nwithan8**
New "Add Credit Card" example for each CL

### CODE REVIEW

File: official/docs/java/current/billing/create-ep-credit-card.java [hunk 1]
```diff
@@ -0,0 +1,17 @@
+package referral;
```
Comment (jchen293) [inline official/docs/java/current/billing/create-ep-credit-card.java:1]: We might need a code example for V5 as well, at least for now.

Comment (nwithan8) [reply official/docs/java/current/billing/create-ep-credit-card.java:1]: @Justintime50 Thoughts? We also have the v3 of C#, didn't know if we were backporting our examples

Comment (jchen293) [reply official/docs/java/current/billing/create-ep-credit-card.java:1]: I think we need one before we officially release the V6 Java

Comment (Justintime50) [reply official/docs/java/current/billing/create-ep-credit-card.java:1]: Yes, these should have been there to begin with and somehow got missed so we'll need to "backport" these examples because the functionality existed in those older versions.

Comment (nwithan8) [reply official/docs/java/current/billing/create-ep-credit-card.java:1]: Resolved

File: official/docs/csharp/v3/billing/create-ep-credit-card.cs [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    {
+        public static async Task Main()
+        {
+            string apiKey = Environment.GetEnvironmentVariable("EASYPOST_API_KEY")!;
+
+            EasyPost.ClientManager.SetCurrent(apiKey);
+
+            string referralUserApiKey = Environment.GetEnvironmentVariable("REFERRAL_USER_API_KEY")!;
+
+            PaymentMethod paymentMethod = await Partner.AddCreditCardToUser(referralUserApiKey, "0123456789101234", "01", "2028", "111", PaymentMethod.Priority.Primary);
```
Comment (Justintime50) [inline official/docs/csharp/v3/billing/create-ep-credit-card.cs:19]: All the params need to match the already existing Curl and Response params please, otherwise it looks goofy.

Did we use the auto-generation tool to create these? If not, this is a great first use-case to see what holes we need to fill.

**Review by Justintime50 — CHANGES_REQUESTED** [review]

File: official/docs/csharp/v3/billing/create-ep-credit-card.cs (hunk 2 above)
Comment (nwithan8) [reply official/docs/csharp/v3/billing/create-ep-credit-card.cs]: Resolved

**Review by Justintime50 — COMMENTED** [review]
The credit card number still doesn't match the responses `last4`

**Review by jchen293 — APPROVED** [review]

**Comment by Justintime50** [comment]
> > The credit card number still doesn't match the responses `last4`
> 
> All credit card API calls are using a Visa card, expiration 01/2025, number 0123456789101234. The only thing that does not use this is a Mastercard in the billing response that is that secondary card, and therefore a different number.

Ok, then let's update the response to match this new number. 

TLDR: I don't really care, as long as everything lines up.

**Comment by nwithan8** [comment]
> > > The credit card number still doesn't match the responses `last4`
> > 
> > 
> > All credit card API calls are using a Visa card, expiration 01/2025, number 0123456789101234. The only thing that does not use this is a Mastercard in the billing response that is that secondary card, and therefore a different number.
> 
> Ok, then let's update the response to match this new number.
> 
> TLDR: I don't really care, as long as everything lines up.

It is.

**Review by Justintime50 — APPROVED** [review]
